app/src/androidTest/java/ch/epfl/sdp/drone3d/ui/map/MissionInProgressActivityIntendedTest.kt

```diff
@@ -231,11 +234,12 @@ class MissionInProgressActivityIntendedTest {
 
         Thread.sleep(500)
 
-        Intents.intended(
-            IntentMatchers.hasComponent(
-                ComponentNameMatchers.hasClassName(ItineraryShowActivity::class.java.name))
+        ViewMatchers.assertThat(
+            activityRule.scenario.state.toString(),
+            Matchers.equalTo(Lifecycle.State.DESTROYED.toString())
```

Comment:
I don't think the toString() are needed. And isn't an activity supposed to be launched ?

Reply:
Ok, I read the rest of the code. No activities should be launched